### DISCUSSION TITLE
4.1.0 on SGI Irix

### DISCUSSION THREAD

**Discussion by dcecchino** [XEphem/XEphem#66] [Show and tell]
Compiled 4.1.0 on SGI Irix 6.5.2, love the compatibility still!

![359061BF-5601-4BE0-8A1E-8F726FF91A42](https://user-images.githubusercontent.com/17807052/204630224-95db9981-a0c0-491b-b7f1-a7e9e80995a8.jpeg)

**Comment by brandon-rhodes** [comment]
Thank you for the screenshot! I'm happy to see XEphem still able to run on an Irix—when I was in graduate school, those workstations were the most prestigious that were available in the College of Computing labs.
